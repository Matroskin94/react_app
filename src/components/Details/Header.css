.title {
    display: inline-block;
}

.favoriteButton {
    display: inline-block;
    position: absolute;
    margin: 50px 20px;
    right: 0px;
}

@media(--mediumScreen) {
    .favoriteButton {
        margin-right: 50px;
    }
}