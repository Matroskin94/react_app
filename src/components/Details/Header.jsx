import React from 'react';

const Header = props => (
    <div>
        <h1>Details Header</h1>
    </div>
);

export default Header;
