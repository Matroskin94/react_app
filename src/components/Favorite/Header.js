import React from 'react';

const Header = props => ({
    render: function() {
        return (
            <div>
                <h3 className = 'textCenter'>Favourites</h3>
            </div>
        );
    }
});

export { Header };
