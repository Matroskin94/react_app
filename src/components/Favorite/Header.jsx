import React from 'react';

const Header = props => (
    <div>
        <h3 className='textCenter'>Favourites</h3>
    </div>
);

export default Header;
