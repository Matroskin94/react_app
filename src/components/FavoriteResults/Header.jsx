import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
    <div data-test='favHeader'>
        <Link to='/' data-test='favesBackLink'> BACK </Link>
        <h3 className='textCenter'>Favourites</h3>
    </div>
);

export default Header;
