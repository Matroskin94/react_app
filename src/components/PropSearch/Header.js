import React from 'react';

const Header = props => ({
    render: function() {
        return (
            <div>
                <button className='button' id='faves-but'>Faves</button>
                <h3>Property cross</h3>
            </div>
        );
    }
});

export { Header };
