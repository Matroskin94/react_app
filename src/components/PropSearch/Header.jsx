import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
    <div data-test='searchHeader'>
        <Link to='/favorite' data-test='favesLink'>Faves </Link>
        <h3>Property cross</h3>
    </div>
);

export default Header;
