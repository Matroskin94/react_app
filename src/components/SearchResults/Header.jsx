import React from 'react';
import PropTypes from 'prop-types';
import ResultsHeader from './ResultsHeader.jsx';
import LoadingHeader from './LoadingHeader.jsx';

const Header = props => {
    const headerText = props.isLoading ?
        <LoadingHeader /> :
        <ResultsHeader
            queries={props.queries}
            address={props.address}
            currentPage={props.currentPage}
            isResultsEmpty={props.isResultsEmpty}
        />;

    return (
        <div>
            {headerText}
        </div>
    );
};

Header.propTypes = {
    queries: PropTypes.array,
    address: PropTypes.object,
    currentPage: PropTypes.number,
    isResultsEmpty: PropTypes.bool,
    isLoading: PropTypes.bool
};

Header.defaultProps = {
    queries: [],
    address: {},
    currentPage: 1,
    isResultsEmpty: true,
    isLoading: true
};

export default Header;
