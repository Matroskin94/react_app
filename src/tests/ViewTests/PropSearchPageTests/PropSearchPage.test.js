import React from 'react';
import { Link } from 'react-router-dom';

import Header from '../../../components/PropSearch/Header.jsx';

describe('PropSearch page testing', () => {
	it('Header test', () => {
		const wrapper = shallow(<Header />);

    	expect(wrapper).toMatchSnapshot();
	});
});