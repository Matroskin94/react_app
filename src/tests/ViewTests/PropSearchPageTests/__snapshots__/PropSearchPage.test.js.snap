// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropSearch page testing Header test 1`] = `
<div
  data-test="searchHeader"
>
  <Link
    data-test="favesLink"
    replace={false}
    to="/favorite"
  >
    Faves 
  </Link>
  <h3>
    Property cross
  </h3>
</div>
`;
